plugins/headers.rst

```diff
@@ -0,0 +1,143 @@
+Header Plugins
+==============
+
+Header plugins are useful to manage request headers. Many operations are possible:
+
+Default headers values
+----------------------
+
+The plugin Http\Plugins\HeaderDefaultPlugin allows to set default values for given headers.
```

Comment:
Indeed it can feel weird for user, and the FQCN is specified in the code example